### DISCUSSION TITLE
Support for importing ISC DHCP Lease Files dhcpd.leases

### DISCUSSION THREAD

**Discussion by skycubeuk** [netalertx/NetAlertX#194] [Ideas]
I can see that there is already a way to read dnsmasq lease files. I was wondering  how difficulte it would be to add support for ISC DHCP Lease Files. As far as I know, it is the most widely used DHCP server.

**Comment by jokob-sk** [comment]
Hi There,

I implemented a plugin supporting imports of multiple DHCP.leases files. It's still to be released, but you can test the dev build if interested:

`docker pull jokobsk/pi.alert_dev:sha-7f25672`

Simple docs:

https://github.com/jokob-sk/Pi.Alert/tree/main/front/plugins/dhcp_leases
